Trappist/src/Promact.Trappist.Repository/Questions/QuestionRepository.cs

```diff
@@ -59,23 +56,26 @@ public QuestionRepository(TrappistDbContext dbContext, UserManager<ApplicationUs
         }
 
         /// <summary>
-        /// Add new code snippet Question to the database
+        /// Adds new code snippet question to the database
         /// </summary>
-        /// <param name="questionAC">Question data transfer object</param>
-        public async Task AddCodeSnippetQuestionAsync(QuestionAC questionAC)
+        /// <param name="questionAC">QuestionAC class object</param>
+        /// <param name="userEmail">Email of logged in user</param>
+        public async Task AddCodeSnippetQuestionAsync(QuestionAC questionAC, string userId)
         {
-            var codeSnippetQuestion = questionAC.CodeSnippetQuestion;
+            var codeSnippetQuestion = Mapper.Map< CodeSnippetQuestionAC, CodeSnippetQuestion>(questionAC.CodeSnippetQuestion);
             var question = Mapper.Map<QuestionDetailAC, Question>(questionAC.Question);
+            //question.CreatedByUser = await _userManager.FindByEmailAsync(userEmail);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS125)